### DISCUSSION TITLE
Change drag zone

### DISCUSSION THREAD

**Discussion by diegoulloao** [react-grid-layout/react-grid-layout#1766] [Q&A] [answered]
It's possible to change the drag zone? Let's say I have a Card component inside the Grid. This Card component has a header and body. I would like to drag the element only dragging it from the header zone and not from the body.

It's that possible?

**Comment by STRML** [comment, marked as answer]
Yes - `draggableHandle` prop.

**Comment by diegoulloao** [comment]
I figured out that I can actually pass the classname to the `draggableHandle` Layout Component prop.

```jsx
<GridLayout draggableHandle=".draggable-element-class" {...props}>
  {/* content */}
</GridLayout>
```
